fastapi/serverless.yaml

```diff
@@ -0,0 +1,67 @@
+frameworkVersion: ">=2.66.0 <3.0.0"
+
+service: fastapi-blueprint
+
+provider:
+  name: aws
+  runtime: python3.9
```

Comment:
Deploy med Serverless Framework funker ikke med 3.9 på Mac pga. manglende `lambci` Docker image: https://github.com/lambci/lambci/issues/138

Kan vi holde oss på 3.8, eller bruker vi features fra 3.9?

Reply:
Å ja, det var det der ja... Holder det at man kan deploye manuelt fra brancher via GitHub Actions i mellomtiden frem til Mac har fått støtte for det, tror du? :sweat_smile: